wordpress-mobile/WordPress-iOS
# Adds GitHub helper script

This PR adds a first release of the `Scripts/github-helper.rb`script that wraps some API calls into commands that are related to common release related tasks.

The PR also introduce a `README.md` file for the Scripts folder, scoped to provide an overview of the scripts, requirements, installation steps and basic usage. 
This first version contains only info about the new script. Plan is to expand coverage with future PRs during the ongoing scripts review efforts. 

To test:
 - Install the Octokib libray: `gem install octokit`
 - Create the `.netrc`file with login info
 - run each command ( `list_open`, `set_prot`, `set_frozen`, `get_frozen`), check that no error is raised and that the result matches the actual repository status. Check README.md for more info about the commands.  

cc/ @elibud @diegoreymendez
